web/.eslintrc

```diff
@@ -30,6 +30,13 @@
         }
       }
     ],
+    // foo(undefined) is not always equivalent to foo()
+    "unicorn/no-useless-undefined": [
+      "error",
+      {
+        "checkArguments": false
+      }
+    ],
```

Comment:
I think we don't need this anymore as we removed the undefined stuff by using `Option` right?

Reply:
b219db2896ebce6806b93a06b58b2d14630dfe5a